CONTRIBUTING.md

```diff
@@ -86,10 +86,34 @@ component of the Thanos distributed system. We recommend:
 `make help` will print most of available commands with relevant details.
 * Spin up a prebuilt dev environment using Gitpod.io
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/thanos-io/thanos)
+* In case you want to develop the project locally, install **Golang** in your machine. Here is a nice [gist](https://gist.github.com/nikhita/432436d570b89cab172dcf2894465753) for this purpose.
+
+### Installing Project locally in your machine
+
+* Change your directory to the gopath for installing Thanos there -
+  * `cd $GOPATH/src/github.com/`
```

Comment:
Hm, you don't need any of this anymore. With Go modules it's ok to clone it wherever.

What about this flow:
1. Find any directory in your system your want Thanos repo in. e.g  `~/Repos`
2. Run `git clone git@github.com:thanos-io/thanos.git` 
3. Make sure you have `GOBIN`, `GOPATH` and `GOBIN` (useful) set to PATH. For example:

```
export GOBIN="~/Repos/thanos/.bin" # It's nice to have local tooling installed and stored locally.
export GOPATH="~/Repos/thanosgopath" # Use if you want to have an isolated directory for deps, otherwise, the dir where you have installed golang will be used.
export GOPROXY="https://proxy.golang.org"
export PATH="$GOBIN:$GOROOT/bin:$PATH"
```

4. Consider persisting variables
5.Done! (make help, make build etc)

Reply:
Sure, made those changes